### DISCUSSION TITLE
1주차 과제 질문

### DISCUSSION THREAD

**Discussion by fkdldjs1016** [rinthel/opengl_course#2] [질문 & 답변]
CMake Configure을 누르면 "error MSB6003: 지정한 작업 실행 파일 "cmd.exe"을(를) 실행할 수 없습니다."라는 오류가 나는데 해결을 못 해 질문드립니다.

![오류화면](https://user-images.githubusercontent.com/80034815/110072394-10dcd480-7dc1-11eb-80f3-0db2ae0fe3b3.png)
[오류내용.txt](https://github.com/rinthel/opengl_course/files/6088332/default.txt)

**Comment by rinthel** [comment]
@fkdldjs1016 윈도우 계정명이 한글이라서 생기는 문제로 의심됩니다.
윈도우에서 사용하는 컴파일 도구들은 한글 경로를 잘 지원하지 못할때가 있거든요.
- 윈도우에 영문으로된 계정을 하나 더 추가하여 다시 시도해보거나,
- 프로젝트 디렉토리를 `c:\Users\박지민` 이외의 곳에 (예: `c:\dev\`) 옮긴 뒤 다시 시도해보세요

**Reply by rinthel** [reply]
@fkdldjs1016 혹시 해결이 됐으면 답글 하나 달아주세요-

**Reply by fkdldjs1016** [reply]
> @fkdldjs1016 혹시 해결이 됐으면 답글 하나 달아주세요-

해결 됐습니다. 감사합니다.
